docs/proton-kafka.md

```diff
@@ -685,13 +696,3 @@ batch.size                               |  P  | 1 .. 2147483647 |       1000000
 delivery.report.only.error               |  P  | true, false     |         false | low        | Only provide delivery reports for failed messages.  *Type: boolean*
 sticky.partitioning.linger.ms            |  P  | 0 .. 900000     |            10 | low        | Delay in milliseconds to wait to assign new sticky partitions for each topic. By default, set to double the time of linger.ms. To disable sticky behavior, set to 0. This behavior affects messages with the key NULL in all cases, and messages with key lengths of zero when the consistent_random partitioner is in use. These messages would otherwise be assigned randomly. A higher value allows for more effective batching of these messages.  *Type: integer*
 
-
-## Limitations
```

Comment:
for a long document like this, I guess people still expect to see a list of limitations or known issues to avoid doing something wrong. For example, many times people ask whether consumer group is used in Kafka external stream. They may not read docs, but with docs to explain this, this is a good reference or AI can answer for them

Reply:
These are not limitations from my view

Reply:
yes, FAQ in this page may work better. we can still keep a long FAQ (maybe refer to that long FAQ from this page)